setup/install/environment.md

```diff
@@ -67,6 +67,54 @@ hal config deploy edit --type distributed --account-name $ACCOUNT
 This command changes the type of the next deployment of Spinnaker, and will
 deploy it to the account you have previously configured.
 
+## Local Git
+
+The __Local Git__ installation means Spinnaker will be cloned, built and run on
+the single machine Halyard is run on.
+
+### Intended Use-case
+
+The __Local Git__ installation is intended for developers who want to contribute
+to Spinnaker. It is not intended to be used to manage any production environment.
+
+### Prerequisites
+
+#### Install local dependancies
+
+Ensure that the following are installed on your system
+
+* git
+* curl
+* yarn ([guide](https://yarnpkg.com/lang/en/docs/install/))
+
+#### Fork all Spinnaker repos
+
+Fork all of the microservices listed here: [Spinnaker Microservices](https://www.spinnaker.io/reference/architecture/#spinnaker-microservices) on github ([guide](https://guides.github.com/activities/forking/#fork)).
+
+#### Setup SSH Keys
+
+Follow the following guides to setup ssh access to your github.com account from your local machine.
+
+* [Generating a new ssh key and adding it to your ssh agent](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
+* [Adding a new ssh key to your Github account](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/)
+
+#### Connect to Redis
+
+Either:
+
+1. open a tunnel on 6379, or
+2. run a local instance
+
+### Required Hal Invocations
+
+Currently, Halyard defaults to a __Local Debian__ install when first run, so 
+Developers must change their deployment type to __Local Git__ type. You can run 
+the following command.
+
+```
+hal config deploy edit --type localgit --git-origin-user=<YOUR_GITHUB_USERNAME>
```

Comment:
I'd point out that `YOUR_GITHUB_USERNAME` is where stuff got forked to above.

Reply:
Done.